docs/examples/ds-ig.yaml

```diff
@@ -0,0 +1,57 @@
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  name: ig
+  labels:
+    k8s-app: example-ig
+spec:
+  selector:
+    matchLabels:
+      name: example-ig
+  template:
+    metadata:
+      labels:
+        name: example-ig
+    spec:
+      containers:
+      - name: ig
+        # Replace this with your own image
+        image: alpine:3.17
+        securityContext:
+          privileged: true
```

Comment:
Indicate it's possible to run without privileged and point to our deploy.yaml with details about capabilities needed?